src/quantum/azext_quantum/commands.py

```diff
@@ -64,7 +64,7 @@ def one(key, value):
         return OrderedDict([
             ('Result', key),
             ('Frequency', f"{value:10.8f}"),
-            ('', f"\u2590{barra:<22} |"),
+            ('', f"\u007C{barra:^20}\u007C")  # f"\u23B9{barra:^22}\u23B8")
```

Comment:
@bettinaheim , is there a reason why we are drawing the histogram this way?
Also, is it intentional to leave the comment with U+23B9 and U+23B8 or is it there because of experimentation?

Reply:
The bars for the simulator looked confusing to me because you often get a bar of 100%, but as it was, it didn't look like the bar was entirely filled up (it has no gap on the left side, and then a gap before the final dash on the right side, if I remember correctly). Ideally, I would have made it so that the bar just looks full, but for reasons I can't remember now I couldn't get that with the characters that looked like they were commonly supported in the terminal while also keeping a bar on the right to indicate where the 100% would sit. I am happy to adjust if you have other suggestions. The comment I believe was the better option in character choice but didn't look like it was commonly supported. I removed it.